src/track/cue.h

```diff
@@ -48,14 +48,33 @@ class Cue : public QObject {
 
     double getEndPosition() const;
 
+    void readInfo(
+            mixxx::AudioSignal::SampleRate sampleRate,
+            mixxx::CueInfo* pCueInfo,
+            bool* pDirty = nullptr) const;
```

Comment:
Where are you planning to use this function?

Reply:
aoide export to get rid of the redundant conversion that should be done here

Reply:
Having both directions of the conversion implemented at a single place is preferred.

Reply:
Agreed. We don't need to repeat this pattern here.